mappings/net/minecraft/client/render/entity/LivingEntityRenderer.mapping

```diff
@@ -33,7 +38,11 @@ CLASS net/minecraft/class_2366 net/minecraft/client/render/entity/LivingEntityRe
 	METHOD method_4016 getEntityAge (Lnet/minecraft/class_1752;F)F
 		ARG 1 entity
 		ARG 2 tickDelta
-	METHOD method_9652 (Lnet/minecraft/class_871;)V
+	METHOD method_9651 angleRotated (FFF)F
```

Comment:
Methods should generally be verbs (`rotateAngle` or from the arg names it looks like an interpolation, so `interpolateAngle`?).